packages/core/src/components/sections/ReviewsAndRatings/section.module.scss

```
@@ -1,7 +1,49 @@
 @layer components {
   .section {
+    @import "@faststore/ui/src/components/atoms/Icon/styles.scss";
+    @import "@faststore/ui/src/components/atoms/Link/styles";
+    @import "@faststore/ui/src/components/atoms/Select/styles";
     @import "@faststore/ui/src/components/molecules/Rating/styles";
+    @import "@faststore/ui/src/components/molecules/SelectField/styles";
     @import "@faststore/ui/src/components/organisms/RatingSummary/styles";
-    @import "@faststore/ui/src/components/atoms/Button/styles";
+
+    [data-fs-content] {
```

after updating [here](https://github.com/vtex/faststore/pull/2713/files#r2007960455) just to make it clear.
```suggestion
    [data-fs-content="review-and-ratings"] {
```